Login.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const response = await axios.post('/api/login', { email, password });
+      if (response.data.success) {
+        window.location.href = "/home";
+      } else {
+        alert(response.data.message);
+      }
```

Comment:
Avoid using alerts for displaying error messages. They are blocking and not user-friendly. Consider using a more user-friendly way to display error messages.